api.js

```diff
@@ -69,8 +69,8 @@ function getLocationFromVF(mmsi, cb) {
       const latitude = splitted[0].indexOf('N') === -1 ? parseFloat(splitted[0]) * -1 : parseFloat(splitted[0]);
       const longitude = splitted[1].indexOf('E') === -1 ? parseFloat(splitted[1]) * -1 : parseFloat(splitted[1]);
 
-      const timestamp = new Date($('.vfix-top:nth-of-type(1) .tparams tr:nth-of-type(11) .v3').text()).toString();
-      const unixtime = new Date($('.vfix-top:nth-of-type(1) .tparams tr:nth-of-type(11) .v3').text()).getTime()/1000;
+      const timestamp = new Date($('#lastrep').attr('data-title')).toString();
```

Comment:
The relevant `<td>` tag has the `#lastrep` ID. IDs are unique, so this seems like a more straightforward way of querying for the right element.

From https://www.vesselfinder.com/vessels/FSO-ASIA-IMO-9224752-MMSI-538003426:

```html
<td class="v3 tooltip expand" data-title="Jan 30, 2020 20:56 UTC" id="lastrep">
    <span class="">40 mins ago </span>
    <span class="info"><i>i</i></span>
</td>
```